### ISSUE TITLE
It's not working with MacOS Sierra

### ISSUE THREAD

**Issue by affmarker** [tekezo/Karabiner#732]
I try alot of Change Key function however its not working

**Comment by qiangu** [comment]
I also have this problem, it's not working with mechanical keyboards, F1-F12 functional key can't be changed anyway.

**Comment by fengmiaosen** [comment]
I also have same problem

**Comment by aruis** [comment]
+1

**Comment by 0oneo** [comment]
+1

**Comment by hnn** [comment]
Unfortunately, this seems to be a larger issue, which will take some time to address.

For details on the discussion of this, see [this issue](https://github.com/tekezo/Seil/issues/68) for Seil.

**Comment by yangbryant** [comment]
+1

**Comment by wujekbogdan** [comment]
+1

**Comment by Douglasli** [comment]
+1

**Comment by sasa** [comment]
+1

**Comment by sbarex** [comment]
+1

**Comment by anhdat** [comment]
To people who are going to "+1" on issues. Please use the Reaction button instead. There are people subscribe to this issue, wating for updates. Thanks.

**Comment by jmatth** [comment]
An issue for this already exists: https://github.com/tekezo/Karabiner/issues/660. Sounds like you should try [Karabiner-Elements](https://github.com/tekezo/Karabiner-Elements) if you need something that works right now.

**Comment by anhdat** [comment]
agree with @jmatth. @affmarker can this issue be closed and we will watch on #660 instead?

**Comment by harryzjm** [comment]
+10086

**Comment by Caf** [comment]
It's already discussed [here](https://github.com/tekezo/Karabiner/issues/660), L2search? And donate for this awesome work.

**Comment by steve-jokes** [comment]
+1s

**Comment by skyjia** [comment]
+1

**Comment by redigiovanni** [comment]
Karabiner is absolutely a great tool that I've come to rely on every day.... it's a can't live without it tool once you start using it.... I learned the hard way that Sierra is completely incompatible with it for now,  and feel like an idiot for not checking first like I normally do!  I just got done with the painstakingly  long 6-8 hour procedure to revert back to El Capitan... Ugh!   But it was well worth the effort to get it back.... I'll never forget again to always always always check compatibility issues with any OS X upgrades!
